### DISCUSSION TITLE
Is the API down?

### DISCUSSION THREAD

**Discussion by alee279** [weather-gov/api#718] [General]
I have been getting 502 Bad Gateway all afternoon, but I did not see anything posted about it

**Comment by akrherz** [comment]
The status message is here https://www.nco.ncep.noaa.gov/status/messages/

**Reply by geiseri** [reply]
Database maintenance didn't go well? ;)

**Reply by digamesystems** [reply]
Thanks for the link!

**Comment by aasmith26** [comment]
I also see the 502 errors in my API calls.  Have emailed nco.ops@noaa.gov and they just replied back they are working to mitigate the issue and no ETA of return to service is known at this time.

**Reply by aasmith26** [reply]
Per NCEP the API is back functioning normally again.  As of 0122Z.

**Comment by digamesystems** [comment]
We've had the same 502 issue on the /gridpoints/ API since about 9:00 this morning (Pacific time)

**Comment by digamesystems** [comment]
Thanks. I appreciate the update.
________________________________
